devnotes/scom_access.md

```diff
@@ -0,0 +1,92 @@
+# Access to SCOM registers
```

Comment:
Are we trying to access register(s) from: https://github.com/3mdeb/openpower-coreboot-docs/blob/main/devnotes/register_for_SCOM_tests.md

If so, explain that and link to this document

Reply:
I have been trying to read different register that has other than 0 value on boot. I guess I should add it to the linked document.

Reply:
I have added this register to the document https://github.com/3mdeb/openpower-coreboot-docs/pull/10